### ISSUE TITLE
Docker image missing / broken

### ISSUE THREAD

**Issue by Nold360** [Trow-Registry/trow#351]
**To Reproduce**
Try to install the helm-chart with default values.

1. There is no tag `0.6.0` == appVersion
2. The `latest` tagged image is broken. I built a new image from hand & it worked as expected.

**Comment by luatnd** [comment]
I use :latest and also got:
```
Events:                                                                                               │
│   Type     Reason     Age              From               Message                                     │
│   ----     ------     ----             ----               -------                                     │
│   Normal   Scheduled  12s              default-scheduler  Successfully assigned default/trow-0 to nod │
│ epool-079fbdeb-6a04-4806-a4-node-ffea73                                                               │
│   Normal   Pulling    6s (x2 over 9s)  kubelet            Pulling image "ghcr.io/trow-registry/trow:l │
│ atest"                                                                                                │
│   Normal   Pulled     6s               kubelet            Successfully pulled image "ghcr.io/trow-reg │
│ istry/trow:latest" in 2.86783899s (2.867859124s including waiting)                                    │
│   Normal   Created    5s (x2 over 6s)  kubelet            Created container trow                      │
│   Warning  Failed     5s (x2 over 6s)  kubelet            Error: failed to create containerd task: fa │
│ iled to create shim task: OCI runtime create failed: runc create failed: unable to start container pr │
│ ocess: exec: "-n": executable file not found in $PATH: unknown                                        │
│   Normal   Pulled     5s               kubelet            Successfully pulled image "ghcr.io/trow-reg │
│ istry/trow:latest" in 823.922027ms (823.932676ms including waiting)                                   │
│   Warning  BackOff    4s (x2 over 5s)  kubelet            Back-off restarting failed container trow i │
│ n pod trow-0_default(4a1a81d7-dc58-440d-bee4-5890b0e9939e)      
```

**Comment by Nold360** [comment]
> @Nold360 Can you gimme your image, I could not build it with both build and buildx

sure, it's `nold360/trow`. I needed to fix the Dockerfile a bit to build it usind podman, too.

Did you try building it with docker? could you get me the error message? I'll make a PR ready then

**Comment by awoimbee** [comment]
Hi,
Sorry I missed that the chart would get released but not the container after I upstreamed my changes (https://github.com/Trow-Registry/trow/pull/347). Fixed !
I just reuploaded the existing `ghcr.io/extrality/trow:0.6.0`.

> I could not build it with both build and buildx

For me (locally), `./docker/build.sh` works, but not `./docker/multi-arch.sh` (`error: linker 'clang' not found`). We would be happy to accept PRs improving the build system.
My experience is that cross compiling is always a PITA, even for Rust.

> I needed to fix the Dockerfile a bit to build it usind podman, too

Same, would happily accept a PR !
